output.cpp

```diff
@@ -118,11 +117,44 @@ void msp_output(input_t *input) {
 static void fb_output_haplotype(FILE *f, int16_t *p, int n) {
   fprintf(f,"%1.5f",DF16(p[0]));
   for(int k=1; k < n; k++)
-    fprintf(f," %1.5f",DF16(p[k]));
+    fprintf(f,"\t%1.5f",DF16(p[k]));
 }
 
 #define FB_EXTENSION ".fb.tsv"
 void fb_output(input_t *input) {
+// the output is a tab separated file with the name \<output basename\>.fb.tsv
+//
+// The first line is a comment line, that specifies the order of subpopulations:
+// eg:
+//   #reference_panel_population: golden_retriever  labrador_retriever  poodle  poodle_small
+// The second line specifies the column names, and every following lines gives 
+// data on a chunk of the genome, called a conditional random field (CRF) point.
+// The first few columns specifies the chromosome, genetic marker's physical 
+  // position in basepair units and genetic position in centiMorgans, and the 
+  // genetic marker's numerical index in the rfmix genetic map input file. 
```

Comment:
it's the way sublime text comments outs multiple lines of code with `CMD-/`. i changed to a block comment `/*` `*/`